kore/src/Kore/ASTVerifier/ModuleVerifier.hs

```diff
@@ -123,9 +126,15 @@ newVerifiedModule :: ParsedModule -> Verifier VerifiedModule'
 newVerifiedModule module' = do
     VerifierContext { implicitModule } <- Reader.ask
     let Module { moduleName, moduleAttributes } = module'
+        ImplicitIndexedModule indexedModule = implicitModule
+        emptyIM = emptyIndexedModule moduleName
+    indexedModule' <- traverse
```

Comment:
Do we ever need an `implicitModule` that is _not_ verified? Maybe we could make that module verified from the beginning.